include/vcpkg/base/messages.h

```diff
@@ -2122,6 +2122,12 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "The /{path} file does not exist. This file must exist for CMake helper ports.");
+    DECLARE_MESSAGE(PortBugMissingProvidedUsage,
+                    (msg::spec, msg::value),
+                    "",
+                    "The port provided \"usage\" but forgot to install to\n"
+                    "${{CURRENT_PACKAGES_DIR}}/share/{spec}/usage\n"
```

Comment:
Don't add the path and \n here. Please handle this in the calling code using `LocalizedString::append_raw()`

Reply:
Does the latest commit match your meaning?

Reply:
> Does the latest commit match your meaning?

better, but you just removed `${{CURRENT_PACKAGES_DIR}}` from the path. Please set the entire path in the calling code.